backend/apps/users/models.py

```diff
@@ -13,36 +13,28 @@
 class Roles(models.TextChoices):
     # User Roles
     SUPER_ADMIN = 'SUPER_ADMIN', _('SUPER_ADMIN')
-    ADMIN = 'ADMIN', _('ADMIN')
-    USER = 'USER', _('USER')
+    DOCTOR = 'DOCTOR', _('DOCTOR')
+    PATIENT = 'PATIENT', _('PATIENT')
+    PHARMACY_USER = 'PHARMACY_USER', _('PHARMACY_USER')
 
-
-class Company(SafeDeleteModel):
-    name = models.BinaryField(max_length=100, null=True, blank=True)
-    logo = models.TextField(null=True, blank=True)
-    owner = models.ForeignKey('users.User', on_delete=models.CASCADE,
-                              related_name='managing_company', null=True, blank=True)
-    created_date = models.DateTimeField(auto_now_add=True)
-
-
-class User(AbstractUser, SafeDeleteModel):
+class User(AbstractUser,SafeDeleteModel):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     role = models.CharField(
         max_length=20,
         choices=Roles.choices,
-        default=Roles.USER
+        default=Roles.DOCTOR
```

Comment:
A default role cannot be set according to this scenario. This Role field is always expected to be the choice of the user